sbatches/farmshare/singularity-jupyter.sbatch

```diff
@@ -15,19 +15,21 @@
 # $ bash start.sh singularity /path/to/container <args>
 
 PORT=$1
-NOTEBOOK_DIR=${2:-${SCRATCH}}
-CONTAINER=${3:-docker://vanessa/repo2docker}
+NOTEBOOK_DIR=/farmshare/scratch/users/$USER
+CONTAINER=/farmshare/home/classes/bioe/301p/ce/ce
 
-export SINGULARITY_CACHEDIR=/farmshare/user_data/${USERNAME}/.singularity
```

Comment:
I don't think people will want containers dumped in their home - it's probably better to do .singularity or .singularity/cache (unless the UI has changed recently and I'm not aware?)

Reply:
Changed to put it back in ./singularity as of commit 
024d8bf.

Reply:
Just curious why you are using `$USERNAME` here instead of `$USER` which you just used above?